### PR TITLE
Actor to Artist

### PR DESCRIPTION
**Pull request by Evankhell561**
## About The Pull Request
implements the artist job and removes the actor job.

## Why It's Good For The Game
one more part of the completed documentation.

## Changelog
:cl:
tweak: the actor now works for the Guild.
add: artist bench, an autolate that allows you to create works of art.
add: artist outfit.
tweak: The actor's office has been remodeled to be habitable.
add: perk artist.
add: works of art produced on the artist bench.
/:cl:

### CODE REVIEW

**Comment by TorinoFermic** [comment]
This PR is lacking the use of map merge tool

**Comment by Evankhell561** [comment]
> This PR is lacking the use of map merge tool

I know, what adds the amount of extra lines is this:
image

**Comment by TorinoFermic** [comment]
> > This PR is lacking the use of map merge tool
> 
> I use it, what adds the amount of extra lines is this:
> image

Ah, any reason for keeping the backup ?

**Comment by Evankhell561** [comment]
> > I use it, what adds the amount of extra lines is this:
> > image
> 
> Ah, any reason for keeping the backup ?

well that's the result of using the map marge. I could include it to gitignore

**Comment by TorinoFermic** [comment]
> > > I use it, what adds the amount of extra lines is this:
> > > image
> > 
> > 
> > Ah, any reason for keeping the backup ?
> 
> well that's the result of using the map marge. I could include it to gitignore

Yeah, these files aren't needed.

File: code/modules/sanity/sanity_mob.dm [hunk 1]
```diff
@@ -256,11 +281,11 @@
...
 				if(1 to 25)
 					M.stats.addTempStat(STAT_COG, 5, INFINITY, "Fate Nihilist")
-				if(26 to 50)
+				if(25 to 50)
 					M.stats.removeTempStat(STAT_COG, "Fate Nihilist")
-				if(51 to 75)
+				if(50 to 75)
 					penalty *= -1
-				if(76 to 100)
+				if(75 to 100)
```
Comment (TorinoFermic) [inline code/modules/sanity/sanity_mob.dm:288]: These lines looks suspicious

**Comment by TorinoFermic** [comment]
Should we merge it while this PR contains many lines with /// temporary desc ?

**Review by genessee596 — APPROVED** [review]
I'm the delegate that's been assigned to manage this project.

Evan, I am extremely impressed with both your work and your efficiency. I'm glad to have you on the team.

This looks like it ticks every box that we wanted Artist to do, so I'm cool with merging it.

File: .gitignore [hunk 2]
```diff
@@ -18,4 +18,4 @@ __pycache__
 
 #vscode
 .vscode
-.history
```
Comment (MadmanMartian) [inline .gitignore:21]: how the heck'd you end up in here?

File: code/modules/sanity/sanity_mob.dm (hunk 1 above)
Comment (Evankhell561) [reply code/modules/sanity/sanity_mob.dm:288]: uhm, this is an error that I introduced when coding this, at that time I didn't really know how the switch worked.

**Comment by TheShown911** [comment]
Conflicts

File: code/game/jobs/job/guild.dm [hunk 3]
```diff
@@ -176,3 +176,46 @@ Character Expectations:<br>\
...
 	icon_state = "player-beige"
 	join_tag = /datum/job/mining
+
+/datum/job/artist
+	title = "Guild Artist"
+	flag = ARTIST
+	department = DEPARTMENT_GUILD
+	department_flag = GUILD
+	faction = "CEV Eris"
+	total_positions = 1
```
Comment (TheShown911) [inline code/game/jobs/job/guild.dm:186]: There can be only one (maybe one clown is too limited, I'd recommend 3)

File: code/game/jobs/job/guild.dm [hunk 4]
```diff
@@ -176,3 +176,46 @@ Character Expectations:<br>\
...
+	supervisors = "the Guild Merchant"
+	selection_color = "#dddddd"
+	also_known_languages = list(LANGUAGE_CYRILLIC = 15, LANGUAGE_SERBIAN = 5, LANGUAGE_JIVE = 80)
+	access = list(access_maint_tunnels, access_mailsorting, access_cargo, access_cargo_bot, access_mining, access_mining_station, access_artist, access_theatre)
+
+	outfit_type = /decl/hierarchy/outfit/job/cargo/artist
+	wage = WAGE_LABOUR_DUMB	//Barely a retaining fee. Actor can busk for credits to keep themselves fed
+	stat_modifiers = list(
+		STAT_TGH = 30, //basically a punching bag, he can't robust anyone or shoot guns anyway
+	)
```
Comment (TheShown911) [inline code/game/jobs/job/guild.dm:197]: Comment false now, clowns can shoot guns

File: code/game/machinery/autolathe/artist_bench.dm [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+			//		caliber = pick(CAL_357)
+
+			if("sniper")//From sniper.dm, Arbitrary values
+				R.caliber = CAL_ANTIM
+				R.bulletinsert_sound = 'sound/weapons/guns/interact/rifle_load.ogg'
+				R.fire_sound = 'sound/weapons/guns/fire/sniper_fire.ogg'
+				R.one_hand_penalty = 15 //From sniper.dm, Temporary values
+				R.recoil_buildup = 90
+
+			if("gyro")//From gyropistol.dm, Arbitrary values
```
Comment (TheShown911) [inline code/game/machinery/autolathe/artist_bench.dm:244]: Based?

File: code/game/objects/structures/crates_lockers/closets/secure/cargo.dm [hunk 6]
```diff
@@ -33,3 +33,44 @@
 	new /obj/item/ammo_casing/shotgun/beanbag(src)
 	new /obj/item/ammo_casing/shotgun/beanbag(src)
 	new /obj/item/device/scanner/price(src)
+
+/obj/structure/closet/secure_closet/personal/artist
+	name = "guild artis's locker"
```
Comment (TheShown911) [inline code/game/objects/structures/crates_lockers/closets/secure/cargo.dm:38]: Missed a t here(artist's)

File: code/modules/clothing/masks/gasmask.dm [hunk 7]
```diff
@@ -71,6 +71,31 @@
...
+	var/list/options = list()
+	options["True Form"] = "artist"
+	options["The clown"] = "clown"
+	options["The mime"] = "mime"
+	options["The Feminist"] = "sexyclown"
+	options["The Madman"] = "joker"
+	options["The Rainbow Color"] ="rainbow"
+	options["The monkey"] = "monkeymask"
+	options["The Owl"] = "owl"
+
```
Comment (TheShown911) [inline code/modules/clothing/masks/gasmask.dm:91]: This lost is so cursed, I'd do it the classic way

**Review by TheShown911 — COMMENTED** [review]
Code's mostly clean, due to the size of the PR I'd recommend a test merge on the test server, though it could be tested on the main server.

**Review by genessee596 — APPROVED** [review]

**Comment by genessee596** [comment]
Alright, we're gonna merge this on the last day of public server this week.

That's;
**Monday, November 30th**

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**